src/main/java/org/prebid/server/spring/config/WebConfiguration.java

```diff
@@ -400,4 +404,36 @@ public void startAdminServer() {
             logger.info("Successfully started Admin Server");
         }
     }
+
+    @Configuration
+    @ConditionalOnProperty(prefix = "prometheus", name = "port")
```

Comment:
Please move this configuration to `MetricsConfiguration` as it's more about metrics than PBS web-tier.